platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -7,116 +7,133 @@
   #
   base_url: http://localhost:8085
-
+  # Authentication config the anchor platform server to safely communicate with the business server.
   auth:
-    ## The authentication config for the Anchor server to safely communicate with the Platform server, particularly
-    ## when hosted in different clusters.
-    ##
-    ## @param: type
-    ## @supported_values: none, api_key, jwt
-    ##     none: no authentication is used
-    ##     api_key: The authentication is done using an API key added to the HTTP request header.
-    ##     jwt: The authentication is done using a JWT token added to the `Authorization` header. This token is
-    ##          generated from the secret key.
-    ##
-    ## If the type is `api_key` or `jwt`, the secret must be set by the environment variable `secret.platform_api.auth_secret`.
-    ## If the type is api_key, the secret will be used as the api_key.
-    ## If the type is jwt, the secret will be used as the jwt shared secret.
-    ##
-    ## Secrets are shared on both sides (Platform and Anchor) and are used in CallbackAPI
-    ## requests (`Platform->Anchor`) so the Anchor can ensure the party making the request
-    ## is the Platform
+    #
+    # `type` can be one of: none, api_key, jwt
+    #     none: no authentication is used
+    #     api_key: The authentication is done using an API key added to the HTTP request header.
+    #     jwt: The authentication is done using a JWT token added to the `Authorization` header. This token is
+    #          generated from the secret key.
+    #
+    # If the type is `api_key` or `jwt`, the secret must be set by the environment variable `secret.callback_api.auth_secret`.
+    # If the type is api_key, the secret will be used as the api_key.
+    # If the type is jwt, the secret will be used as the jwt shared secret.
+    #
+    #
+    # Secrets are shared on both sides (anchor platform and business server) and are used in `Platform API`
+    # requests (`business server -> anchor platform`) so the anchor platform can ensure the party making the request
+    # is the business server.
     #
     type: none
+    # The JWT authentication config, if the type is set to `jwt`
     jwt:
+      # The JWT expiration in milliseconds
       expiration_milliseconds: 30000
+      # The HTTP header name of the JWT token
       http_header: Authorization
+    # The API key config, if the type is set to `api_key`
     api_key:
+      # The HTTP header name of the API key
       http_header: X-Api-Key
 
+##############################
+# Payment Observer Configuration
+##############################
 payment_observer:
   # The context path of the Stellar observer server
   context_path: /
   # The listening port of the Stellar observer server
   port: 8083
-  ## @param: type
-  ## @type: string
-  ## @supported_values:
-  ##    `stellar`: the observer will observer transactions from the Stellar network
+  # The management server port
+  management_server_port: 9083
```

Comment:
The management server is the metrics server right? Can it be renamed?

Reply:
The management server is the Java management server exposed by Spring actuator.

Reply:
Does it have swagger or something like that? How would developer know how to use management server?
Also, how is it protected by default?

Reply:
The management port is the port used by Spring actuator. 
https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html

It is used for Prometheus to scrape the metrics.

Reply:
Can you add it in the code comments please?

Reply:
This PR is blocking the implementation. I will add the comments with the impl PR.

Reply:
Sure thank you